collections/_posts/2026/08/02/2026-08-02-refactoring-authorization-at-scale-permission-key-migration.md

```
@@ -0,0 +1,332 @@
+  title: 'Credentials',
+  permissionKey: Keys.SecurityManagementViewCredentials,
+},
+```
+
+---
+
+## The Pipeline, End-to-End
+
+Zero manual work from spreadsheet to rendered UI:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)